beacon-chain/sync/backfill/metrics.go

```diff
@@ -50,13 +71,20 @@ var (
 			Buckets: []float64{50, 100, 300, 1000, 2000},
 		},
 	)
-	backfillBatchTimeDownloading = promauto.NewHistogram(
+	backfillBatchTimeDownloadingBlocks = promauto.NewHistogram(
 		prometheus.HistogramOpts{
-			Name:    "backfill_batch_time_download",
+			Name:    "backfill_batch_blocks_time_download",
 			Help:    "Time batch spent downloading blocks from peer.",
 			Buckets: []float64{100, 300, 1000, 2000, 4000, 8000},
 		},
 	)
+	backfillBatchTimeDownloadingBlobs = promauto.NewHistogram(
+		prometheus.HistogramOpts{
+			Name:    "backfill_batch_blobs_time_download",
+			Help:    "Time batch spent downloading blobs from peer.",
```

Comment:
```suggestion
			Help:    "Time, in milliseconds, batch spent downloading blocks from peer.",
			Buckets: []float64{100, 300, 1000, 2000, 4000, 8000},
		},
	)
	backfillBatchTimeDownloadingBlobs = promauto.NewHistogram(
		prometheus.HistogramOpts{
			Name:    "backfill_batch_blobs_time_download",
			Help:    "Time, in milliseconds, batch spent downloading blobs from peer.",
```

imo good idea to put the units